javascript/javascript3/week1/weather-app/weather.js

```diff
@@ -0,0 +1,137 @@
+const apiKey = "8a65b2856241112124e36f83f5de5e64";
+
+const inputTag = document.getElementById("cityInput");
+const locationBtn = document.getElementById("geolocation");
+const formTag = document.getElementById("search-form");
+
+let map;
+
+const locationData = document.getElementById("location");
+const temperatureData = document.getElementById("temperature");
+const iconDiv = document.getElementById("image");
+const iconData = document.createElement("img")
+iconDiv.appendChild(iconData);
```

Comment:
Why not just do this in the HTML? Would have saved you the extra line.

Reply:
This is an image that I will dynamically populate with a src later. Having <img> tag in HTML without src gave me an error in HTML validator: Error: Element [img] is missing required attribute src.